Automattic/jetpack
# Updates notices / banners for 6.8 release

See screenshots for
- Post-activation banner: #10638
- Post-upgrade banner: #10658


#### Changes proposed in this Pull Request:

* Adds new messaging / images for the post-activation banner and the post-upgrade banner

#### Testing instructions:

1. Spin up a new Jurassic Ninja site with this branch
2. Activate Jetpack and observe the messaging
3. Connect jetpack and observe the messaging
